test/model.test.js

```diff
@@ -6904,7 +6894,7 @@ describe('Model', function() {
       userToUpdate.name = 'John Doe';
 
       // Act
-      const writeOperations = User.buildBulkWriteOperations([newUser, userToUpdate], { timestamps: false, skipValidation: true });
+      const writeOperations = await User.buildBulkWriteOperations([newUser, userToUpdate], { timestamps: false, skipValidation: true });
```

Comment:
`buildBulkWriteOperations` is now sync again, so `await` should likely be removed here and the other places too.
(though technically, `await` on sync is just a no-op; acting as if `await` was not there at all)

Reply:
Fixed in 38c9b18